### PR TITLE
docs: one-off scripts location and docs

### PR DESCRIPTION
**Pull request by amnonkhen**
ebi-ait/hca-ebi-dev-team#420

### CODE REVIEW

**Review by rays22 — APPROVED** [review]
There is a typo (`doen't`), but otherwise looks fine to me.

**Review by aaclan-ebi — APPROVED** [review]
Looks good, I linked the ticket!

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 # Dev Team
 Repository for hca ebi dev team agile management. See [zenhub board](https://app.zenhub.com/workspaces/operations-5fa2d8f2df78bb000f7fb2b5/board?repos=232300832,261790554).
 
+## one-off scripts
+
+Any one-off scripts that doen't have a dedicated location should be maintained in the [scripts](scripts/) directory. See the readme there for more instructions.
```
Comment (ESapenaVentura) [inline README.md:6]:
```suggestion
Any one-off scripts that doesn't have a dedicated location should be maintained in the [scripts](scripts/) directory. See the [readme file](scripts/readme.md) for more instructions.
```

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]
LGTM!
 
Just added a small correction on a typo and a hyperlink to the readme within the scripts folder

**Pull request merged**